src/loadElmStories.tsx

```diff
@@ -25,15 +26,19 @@ export const loadElmStories = (
   name: string,
   module: NodeModule,
   elmApp: { Elm: { Main: any } },
-  storyNames: string[]
+  storyNames: string[],
+  ports: (ports: unknown) => void | null
```

Comment:
```suggestion
  ports?: (ports: unknown) => void | null
```

Let's make this optional. 

Also... is there a better type we can use for `ports`?